project/layersAfterAddingNewLayer.json

```diff
@@ -0,0 +1,162 @@
+[
```

Comment:
The results of adding a new Layer to the existing project mongoDBAtlas testTpen.projects _id 67e5955bdbc0d76c277ff6fc